packages/quantic/force-app/main/default/lwc/quanticPager/e2e/quanticPager.e2e.ts

```diff
@@ -0,0 +1,99 @@
+import {testSearch, testInsight, expect} from './fixture';
+import {useCaseTestCases} from '../../../../../../playwright/utils/useCase';
+
+const numberOfResultsPerPage = 10;
+
+const fixtures = {
+  search: testSearch,
+  insight: testInsight,
+};
+
+useCaseTestCases.forEach((useCase) => {
+  let test = fixtures[useCase.value];
+
+  test.describe(`quantic pager ${useCase.label}`, () => {
+    test.describe('when clicking the next page button', () => {
+      test('should trigger a new search and log analytics', async ({
+        pager,
+        search,
+      }) => {
+        const searchResponsePromise = search.waitForSearchResponse();
+        await pager.clickNextPageButton();
+        const searchResponse = await searchResponsePromise;
+        const {firstResult} = searchResponse.request().postDataJSON();
+        expect(firstResult).toBe(numberOfResultsPerPage);
```

Comment:
FYI you could also do, just if you prefer to deal with objects like this instead of going the de-structuring route.

```suggestion
        expect(
          searchResponse.request().postDataJSON()
        ).toEqual(expect.objectContaining({
          firstResult: numberOfResultsPerPage
        }));
```